build/localnet/manifests/service-accounts.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: debug-client-account
```

Comment:
WHy not specify the namespace for the ServiceAccount too?

Reply:
it's redundant but for consistency let's add it